packages/web/components/templates/homeFeed/HomeFeedContainer.tsx

```diff
@@ -76,17 +77,14 @@ export function HomeFeedContainer(props: HomeFeedContainerProps): JSX.Element {
 
   const gridContainerRef = useRef<HTMLDivElement>(null)
 
-  const [shareTarget, setShareTarget] = useState<LibraryItem | undefined>(
```

Comment:
It looks like there are a bunch of style changes in this that make it a little hard to read and will mess up the git history.